cms/models/pagemodel.py

```diff
@@ -321,6 +321,29 @@ def move_page(self, target, position='first-child'):
         invalidate_cms_page_cache()
         return moved_page
 
+    def revert_to_live(self, language):
+        """Revert the draft version to the same state as the public version
+        """
+        if not self.publisher_is_draft:
+            # Revert can only be called on draft pages
+            raise PublicIsUnmodifiable('The public instance cannot be reverted. Use draft.')
+
+        if not self.publisher_public:
+            raise PublicVersionNeeded('A public version of this page is needed')
```

Comment:
I would rather not, this is an internal. The exception will never be user facing.